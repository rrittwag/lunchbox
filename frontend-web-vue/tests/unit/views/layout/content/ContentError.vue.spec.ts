import ContentError from '@/views/layout/content/ContentError.vue'
import { mount } from '@vue/test-utils'

describe('ContentError', () => {
  test('renders snapshot', () => {
    const wrapper = mount(ContentError)
    expect(wrapper.element).toMatchSnapshot()
  })
})
