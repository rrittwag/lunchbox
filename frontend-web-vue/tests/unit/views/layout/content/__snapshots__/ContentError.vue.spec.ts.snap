// Vitest Snapshot v1

exports[`ContentError > renders snapshot 1`] = `
<h1>
  Error
</h1>
`;

exports[`ContentError renders snapshot 1`] = `
<h1>
  Error
</h1>
`;
